php-checks/src/main/java/org/sonar/php/checks/HardCodedCredentialsInFunctionCallsCheck.java

```diff
@@ -59,8 +61,13 @@ public class HardCodedCredentialsInFunctionCallsCheck extends FunctionArgumentCh
 
   private static final String MESSAGE = "Revoke and change this password, as it is compromised.";
 
-  private static final String JSON_LOCATION = "/org/sonar/php/checks/hardCodedCredentialsInFunctionCallsCheck/sensitiveFunctions.json";
+  private static final String LOCATION_OF_GENERATED_FUNCTIONS_JSON = "/org/sonar/php/checks/hardCodedCredentialsInFunctionCallsCheck" +
+    "/generatedSensitiveFunctions.json";
+  private static final String LOCATION_OF_MANUALLY_CREATED_FUNCTIONS_JSON = "/org/sonar/php/checks" +
+    "/hardCodedCredentialsInFunctionCallsCheck/manuallyCreatedSensitiveFunctions.json";
 
+  private static final Set<String> LOCATION_OF_SENSITIVE_FUNCTIONS_JSON = Set.of(LOCATION_OF_GENERATED_FUNCTIONS_JSON,
+    LOCATION_OF_MANUALLY_CREATED_FUNCTIONS_JSON);
```

Comment:
```suggestion
    private static final String LOCATION_OF_FUNCTIONS_JSON = "/org/sonar/php/checks/hardCodedCredentialsInFunctionCallsCheck/";
  private static final Set<String>SENSITIVE_FUNCTIONS_JSON = Set.of(
    "generatedSensitiveFunctions.json",
    "manuallyCreatedSensitiveFunctions.json");
```
And when parsing the json do `parseResource(LOCATION_OF_FUNCTIONS_JSON + json_location)`